src/main/resources/schema/ans/0.10.11/traits/trait_canonical_url_external.json

```diff
@@ -3,5 +3,6 @@
   "id": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.11/traits/trait_canonical_url_external.json",
   "title": "Canonical External URL",
   "description": "The absolute URL of the document on website which will override any relative URL specified by the `canonical_url` field.",
-  "type": "string"
+  "type": "string",
+  "pattern": "^https?:\\/\\/([a-zA-Z0-9]([a-zA-Z0-9\\-]{0,61}[a-zA-Z0-9])?\\.)+[a-zA-Z]{2,}(\\/[a-zA-Z0-9\\-._~:/%]+)*$"
```

Comment:
I think this forces URL encoding and precludes query strings, since the question mark is omitted, correct?  I think we might see issues with the latter item.  For the former, I'm somewhat conflicted.  Forcing that might be good in many cases.  But, it may be the case that URLs are stored in their unencoded form in other CMSs.  If we take on a client like this, we may not be able to import their canonical URLs without issue.  In the cases where we get this pattern wrong, it would take an ANS version update to fix.

To avoid this situation, it would probably be good to make this pattern more permissive.  Maybe just `^https?://.+$`.  That would avoid the quagmire of making sure we get this pattern just right.  Is there a business case to make this specific that I'm missing?